app/models/flood_risk_engine/enrollment.rb

```diff
@@ -11,7 +11,7 @@ class Enrollment < ActiveRecord::Base
     # we query never from contact to its enrollment
     belongs_to :applicant_contact, class_name: "Contact"
     belongs_to :organisation
-    delegate :org_type, to: :organisation, allow_nil: true
+    delegate :org_type, :partners, to: :organisation, allow_nil: true
```

Comment:
An enrollment does not have partners. Partners are an attribute of Organisation, and only one specific type of Organisation